sass/_prefooter.scss

```diff
@@ -177,7 +194,7 @@
 // Bottom card specific styles
 .bottom-card {
 	grid-column: 1 / -1;
-	background-color: #353946;
+	background-color: #4f5466;
```

Comment:
This is needed because you darken _all_ of the card right? Why not darken right and left but not bottom.